test/quantization/test_da8w4_cpu.py

```diff
@@ -163,6 +163,11 @@ def forward(self, x):
             # ensure the expected op occurs only once in the code after fusion
             # The trailing "(" is to avoid matching the op in the comment
             assert code[0].count("torch.ops.torchao.da8w4_linear_cpu.default(") == 1
+
+            # ensure the custom DA8W4ConcatLinearCPUPass is properly cached as fxgraph
```

Comment:
how does the added test make sure `DA8W4ConcatLinearCPUPass` is properly cached?

Reply:
It checks the counter. Do you think the comment here should be improved? Thanks.

Reply:
yeah, probably some explanation would be helpful, on how the cache works and why the counters would show that `DA8W4ConcatLinearCPUPass` is cached, I don't quite get it right now